src/olympia/activity/tests/test_utils.py

```diff
@@ -24,15 +25,92 @@
 
 
 class TestEmailParser(TestCase):
-
     def test_basic_email(self):
         message = json.loads(open(sample_message).read())
         email_text = message['Message']
         parser = ActivityEmailParser(email_text)
         assert parser.get_uuid() == '5a0b8a83d501412589cc5d562334b46b'
-        assert parser.get_body() == (
+        assert parser.reply == (
             'This is a developer reply to an AMO.  It\'s nice.')
 
+    def test_with_invalid_msg(self):
+        with self.assertRaises(ActivityEmailEncodingError):
+            ActivityEmailParser('youtube?v=dQw4w9WgXcQ')
+
+
+class TestEmailBouncing(TestCase):
+    BOUNCE_REPLY = (
+        'Hello,\n\nAn email was received, apparently from you. Unfortunately '
+        'we couldn\'t process it because of:\n%s\n\nPlease visit %s to leave '
+        'a reply instead; alternatively if you need to send file attachments, '
+        'please reply by email to amo-editors@mozilla.org.\n'
+        '--\nMozilla Add-ons\n%s')
+
+    def setUp(self):
+        self.bounce_reply = (
+            self.BOUNCE_REPLY % ('%s', settings.SITE_URL, settings.SITE_URL))
+
+    @mock.patch('olympia.activity.utils.amo.log')
+    def test_no_note_logged(self, log_mock):
+        message = json.loads(open(sample_message).read())
```

Comment:
curious if we can't just load `sample_message` (here and below) either on import-time or even `setUpClass` to make this a bit less io dependent?